prebid-mobile/pbm-api/android/pbm-adunit-android.md

```diff
@@ -47,12 +47,62 @@ PB Ad Slot is an identifier tied to the placement the ad will be delivered in. T
 
 Trigger a call to Prebid Server to retrieve demand for this Prebid Mobile ad unit.
 
+#### Mopub & GAM
+
+By default, Prebid SDK uses inflection to determine the publisher ad server, one of Mopub or Google Ad Manager (GAM), to convert the Prebid's targeting keys (PBS bid keys, host and cache key) to trigger targeted line items. To render ads in ad servers other than Mopub or GAM, use the next section's 3rd party ad server support feature.
+
+
 **Parameters**
 
 - `adObj`: This is the ad server request object (for [Google Ad Manager](https://developers.google.com/android/reference/com/google/android/gms/ads/doubleclick/PublisherAdRequest) and for [Mopub](https://developers.mopub.com/publishers/reference/android/MoPubView/)). If you do not wish to add any additional /custom key values to the ad server after the Prebid auction, pass `adObj` to the fetchDemand function, where Prebid SDK will set all the Prebid targeting keys as well as any keys added prior to auction
 - As of Prebid SDK 1.7, a publisher can optionally pass the Google Ad Manager `builder` object of the [Google Ad Manager Mobile Ads SDK](https://developers.google.com/android/reference/com/google/android/gms/ads/doubleclick/PublisherAdRequest.Builder) to pass custom keys to Google Ad Manager after the Prebid Auction
 - `onCompleteListener`: listener object
 
+#### 3rd Party Ad Server
+
+As mentioned in the previous section the default ad servers for Prebid SDK are Mopub and GAM. For 3rd part ad server support, the fetchDemand function has been extended to return the Prebid Server bidder key / values (targeting keys) to be passed to the ad server of choice. 
```

Comment:
The default ad servers for Prebid's Mobile SDK are MoPub and GAM. The SDK can be expanded to include support for 3rd party ad servers through the `fetchDemand` function. This function returns the Prebid Server bidder key/values (targeting keys), which can then be passed to the ad server of choice.